test.sh

```diff
@@ -271,8 +271,10 @@ EOF
         echo "Missing AWS credentials file (${aws_creds_path}) and AWS env (AWS_ACCESS_KEY_ID/AWS_SECRET_ACCESS_KEY)"
         exit 1
     fi
+    chmod 500 "${credsfile}"
```

Comment:
This is racy, it should probably be set at creation time done by something like `(umask 077 && mktemp ...)` instead.

Reply:
I have removed the setting of the permissions. This is mounted as `:ro` into the container in any case.

Reply:
I thought the point was to prevent others on the host from reading the file?